lib/api/src/modules/shortcuts.ts

```diff
@@ -205,11 +205,13 @@ export const init: ModuleFn = ({ store, fullAPI }) => {
         }
 
         case 'nextComponent': {
+          event.preventDefault();
```

Comment:
@ghengeveld Is this API public? If so, maybe use `function safePreventDefault(event) { if(event) event.preventDefault() }`?

Reply:
Ah yes, of course.